internal/controller/http/redfish/v1/constants.go

```diff
@@ -0,0 +1,128 @@
+	BaseActionNotSupportedID     = "Base.1.11.0.ActionNotSupported"
+	BaseNoValidSessionID         = "Base.1.11.0.NoValidSession"
+	BaseInsufficientPrivilegeID  = "Base.1.11.0.InsufficientPrivilege"
+	BaseNotAcceptableID          = "Base.1.11.0.NotAcceptable"
+)
+
+// HTTP Header constants for Redfish compliance
+const (
+	ContentTypeJSON       = "application/json; charset=utf-8"
+	ContentTypeHeaderName = "Content-Type"
+	ODataVersionValue     = "4.0"
+	ODataVersionHeader    = "OData-Version"
+	CacheControlValue     = "no-cache"
+	CacheControlHeader    = "Cache-Control"
+	XFrameOptionsHeader   = "X-Frame-Options"
+	XFrameOptionsValue    = "DENY"
+	CSPHeader             = "Content-Security-Policy"
+	CSPValue              = "default-src 'self'"
+)
+
+// Additional HTTP constants
+const (
+	MediaTypeWildcard = "*/*"
+	HeaderAccept      = "Accept"
+)
+
+// HTTP Methods
+const (
+	MethodGET    = "GET"
+	MethodPOST   = "POST"
+	MethodPUT    = "PUT"
+	MethodPATCH  = "PATCH"
+	MethodDELETE = "DELETE"
+)
+
+// Redfish Service Information
+const (
+	RedfishVersion     = "1.11.0"
+	ServiceRootID      = "RootService"
+	ServiceRootName    = "Redfish Root Service"
+	ServiceProduct     = "Device Management Toolkit Console"
+	ServiceVendor      = "Intel Corporation"
+	DefaultServiceUUID = "550e8400-e29b-41d4-a716-446655440000"
+)
+
+// Common Redfish Schema Types - used across multiple files
+const (
+	SchemaServiceRoot                 = "#ServiceRoot.v1_11_0.ServiceRoot"
+	SchemaSessionService              = "#SessionService.v1_0_0.SessionService"
+	SchemaSessionCollection           = "#SessionCollection.SessionCollection"
+	SchemaComputerSystem              = "#ComputerSystem.v1_0_0.ComputerSystem"
+	SchemaComputerSystemCollection    = "#ComputerSystemCollection.ComputerSystemCollection"
+	SchemaSoftwareInventory           = "#SoftwareInventory.v1_3_0.SoftwareInventory"
+	SchemaSoftwareInventoryCollection = "#SoftwareInventoryCollection.SoftwareInventoryCollection"
+	SchemaTask                        = "#Task.v1_6_0.Task"
+	SchemaIntelOEM                    = "#Intel.v1_0_0.Intel"
+)
+
+// Common Redfish API Paths - used across multiple files
+const (
+	PathRedfishRoot            = "/redfish/v1/"
+	PathSystems                = "/redfish/v1/Systems"
+	PathSessionService         = "/redfish/v1/SessionService"
+	PathSessionServiceSessions = "/redfish/v1/SessionService/Sessions"
+	PathTaskService            = "/redfish/v1/TaskService/Tasks/"
+	PathMetadata               = "/redfish/v1/$metadata"
+)
+
+// OData Context paths for metadata
+const (
+	ODataContextTask                        = "/redfish/v1/$metadata#Task.Task"
+	ODataContextSoftwareInventory           = "/redfish/v1/$metadata#SoftwareInventory.SoftwareInventory"
+	ODataContextSoftwareInventoryCollection = "/redfish/v1/$metadata#SoftwareInventoryCollection.SoftwareInventoryCollection"
+)
+
+// Media Type constants
+const (
+	MediaTypeJSON = "application/json"
```

Comment:
MIME types can be used from Gin - see: https://pkg.go.dev/github.com/gin-gonic/gin@v1.11.0/binding#MIMEJSON